### PR TITLE
Update foreman_salt to 2.0.1

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

**Comment by domcleal** [comment]
Could you have a go at adding the API doc precompilation, copying 9e446aff7a7f8a579cb7c021fe0760779b6659c3?

File: rubygem-foreman_salt/rubygem-foreman_salt.spec [hunk 1]
```diff
@@ -78,10 +79,13 @@ cat <<GEMFILE > %{buildroot}%{foreman_bundlerd_dir}/%{gem_name}.rb
 gem '%{gem_name}'
 GEMFILE
 
+foreman_precompile_plugin -a
```
Comment (domcleal) [inline rubygem-foreman_salt/rubygem-foreman_salt.spec]: missing %

File: rubygem-foreman_salt/rubygem-foreman_salt.spec [hunk 2]
```diff
@@ -39,6 +39,7 @@ BuildRequires: %{?scl_prefix}ruby(release)
 %else
 BuildRequires: %{?scl_prefix}ruby(abi) >= %{rubyabi}
 %endif
+BuildRequires: foreman-plugin >= 1.8.0
```
Comment (domcleal) [inline rubygem-foreman_salt/rubygem-foreman_salt.spec]: You need to add a BR for deface.

Comment (stbenjam) [inline rubygem-foreman_salt/rubygem-foreman_salt.spec]: Added

**Comment by domcleal** [comment]
Thanks @stbenjam!  Building for nightly and 1.8 now.

**Pull request merged**